From 8d25ce5712490795c10eeb233dfd70a66771eace Mon Sep 17 00:00:00 2001
Message-Id: <8d25ce5712490795c10eeb233dfd70a66771eace.1423172493.git.feitong.yi@intel.com>
In-Reply-To: <18bb7324f916bffbc75e07ea0ffd3e6e68e7b6d8.1423172493.git.feitong.yi@intel.com>
References: <18bb7324f916bffbc75e07ea0ffd3e6e68e7b6d8.1423172493.git.feitong.yi@intel.com>
From: Deepak M <m.deepak@intel.com>
Date: Sat, 17 Jan 2015 13:08:14 +0530
Subject: [PATCH 19/29] MUST_REBASE [VPG}: intel/core/dsi: Adding missing
 license

The missing licence is added to the file gen_dsi_pipe.c

For: GMINL-5213
Change-Id: I22483b078da6704627c5c86053a70a30d6063e43
Signed-off-by: Deepak M <m.deepak@intel.com>
---
 .../video/adf/intel/core/common/dsi/gen_dsi_pipe.c |   13 +++++++++++--
 1 file changed, 11 insertions(+), 2 deletions(-)

diff --git a/drivers/video/adf/intel/core/common/dsi/gen_dsi_pipe.c b/drivers/video/adf/intel/core/common/dsi/gen_dsi_pipe.c
index ced809e..06b530d 100644
--- a/drivers/video/adf/intel/core/common/dsi/gen_dsi_pipe.c
+++ b/drivers/video/adf/intel/core/common/dsi/gen_dsi_pipe.c
@@ -1,8 +1,17 @@
 /*
- * gen_dsi_pipe.c
+ * Copyright (C) 2014, Intel Corporation.
+ *
+ * This software is licensed under the terms of the GNU General Public
+ * License version 2, as published by the Free Software Foundation, and
+ * may be copied, distributed, and modified under those terms.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
  *
  *  Created on: September 1, 2014
- *      Author: Shobhit Kumar <shobhit.kumar@intel.com>
+ *  Author: Shobhit Kumar <shobhit.kumar@intel.com>
  */
 
 #include <drm/drm_mode.h>
-- 
1.7.9.5

